dashboard/test/controllers/plc/enrollment_evaluations_controller_test.rb

```diff
@@ -12,36 +12,43 @@ class Plc::EnrollmentEvaluationsControllerTest < ActionController::TestCase
     #the same for everyone. Not all users taking an examination will answer the exact same questions.
     @course = create(:plc_course)
     @course_unit = create(:plc_course_unit, plc_course: @course)
-    @module1 = create(:plc_learning_module, name: 'Getting thrown off cliffs', plc_course_unit: @course_unit, module_type: Plc::LearningModule::PRACTICE_MODULE)
-    @module2 = create(:plc_learning_module, name: 'Advanced Ornithology', plc_course_unit: @course_unit, module_type: Plc::LearningModule::CONTENT_MODULE)
-    @module3 = create(:plc_learning_module, name: 'Answering questions honestly', plc_course_unit: @course_unit, module_type: Plc::LearningModule::PRACTICE_MODULE)
-    @module4 = create(:plc_learning_module, name: 'Admitting ignorance', plc_course_unit: @course_unit, module_type: Plc::LearningModule::CONTENT_MODULE)
 
+    @module_cliffs = create(:plc_learning_module, name: 'Getting thrown off cliffs', plc_course_unit: @course_unit, module_type: Plc::LearningModule::CONTENT_MODULE)
```

Comment:
Is this learning about surviving the throw, or it just signing up to actually be thrown? ;)

Great test module names.